conf/profiles/base.config

```
@@ -28,7 +28,7 @@ process {
         time = { 3.hour * task.attempt }
     }
     withLabel:'large' {
-        cpus = { 2 * task.attempt }
+        cpus = 1
```

Follow the same "standard" as other processes:

```suggestion
        cpus = { 1 * task.attempt }
```

---

aaah, ok! I understood now the change you wanted... I thought the idea was to not have 'retries' to RUN_PRINTS, exactly 1. Got it now o/